spec/models/product_import/inventory_reset_strategy_spec.rb

```diff
@@ -4,53 +4,58 @@
   let(:inventory_reset) { described_class.new(excluded_items_ids) }
 
   describe '#reset' do
-    let(:supplier_ids) { enterprise.id }
-    let(:enterprise) { variant.product.supplier }
-    let(:variant) { create(:variant) }
-
-    let!(:variant_override) do
-      create(
-        :variant_override,
-        count_on_hand: 10,
-        hub: enterprise,
-        variant: variant
-      )
-    end
 
     context 'when there are excluded_items_ids' do
+      let(:enterprise) { variant.product.supplier }
+      let(:variant) { build_stubbed(:variant) }
+      let!(:variant_override) do
+        build_stubbed(
+          :variant_override,
+          count_on_hand: 10,
+          hub: enterprise,
+          variant: variant
+        )
+      end
       let(:excluded_items_ids) { [variant_override.id] }
 
       context 'and supplier_ids is []' do
-        let(:supplier_ids) { [] }
         let(:relation) do
           instance_double(ActiveRecord::Relation, update_all: true)
         end
 
         before { allow(VariantOverride).to receive(:where) { relation } }
 
         it 'does not update any DB record' do
-          inventory_reset.reset(supplier_ids)
+          inventory_reset.reset([])
           expect(relation).not_to have_received(:update_all)
         end
       end
 
       context 'and supplier_ids is nil' do
-        let(:supplier_ids) { nil }
```

Comment:
I find it easier to understand the test setup like this. Without a meaningful var name it's pretty hard to infer that the argument of `#reset` is the list of supplier ids and the context of the test.

Reply:
Good point. Let me update the specs.

Reply:
Done.